src/main/java/org/jenkinsci/plugins/configfiles/buildwrapper/ManagedFile.java

```diff
@@ -133,14 +133,13 @@ public HttpResponse doCheckFileId(StaplerRequest req, @AncestorInPath Item conte
             // allowed to you
             if (context != null) {
                 context.checkPermission(Item.CONFIGURE);
+                final Config config = ConfigFiles.getByIdOrNull(context, fileId);
+                if (config != null) {
+                    return ConfigFileDetailLinkDescription.getDescription(req, context, fileId);
```

Comment:
That doesn't seem like the correct return type for this method.  Shouldn't it be returning a FormValidation?
```suggestion
                    return ConfigFileDetailLinkDescription.getDescription(req, context, fileId);
```

Reply:
I agree. This return is odd, but was already there

Reply:
In fact, it's an implementation of `HttpResponse`. It's encapsulating a message with a link to the file, see https://github.com/jenkinsci/config-file-provider-plugin/blob/master/src/main/java/org/jenkinsci/plugins/configfiles/utils/ConfigFileDetailLinkDescription.java

Reply:
You're right.  Thanks for the correction!